en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+# Choreo Expression Syntax
+
+Choreo expressions comprise a powerful expression syntax based on the [Ballerina language](https://ballerina.io/) for function invocations, declaring a variable using literals and tuples, typecasting, etc. This document presents you the common expressions that you can use while developing Choreo applications.
+
+## Supported Literals 
+
+### String
+
+A string is simply a sequence of characters. It is required to use double quotes (")  to mark the boundary between the starting and ending characters.
+
+#### Examples
+
+  ```
```

Comment:
Shall we use ballerina code formatting for all the samples (i.e. "```ballerina") So it will give proper syntax highlighting.

Reply:
done